starwars/src/App.js

```diff
@@ -33,6 +35,11 @@ class App extends Component {
     return (
       <div className="App">
         <h1 className="Header">React Wars</h1>
+        <div className="character-list">
+            {this.state.starwarsChars.map(starWarsFromMap => (
```

Comment:
Excellent syntax, only I would recommend to use something less verbose for `starWarsFromMap` such as `character`